pkgs/applications/misc/xmenu/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchgit, libX11, libXft, imlib2 }:
+
+{
+  pname = "xmenu";
+  version = "2020-07-23";
+
+  src = fetchgit {
```

Comment:
Can you use `fetchFromGitHub` here? It’s more efficient as it doesn’t have to do a full git clone.